auditor/src/client/mod.rs

```diff
@@ -170,6 +176,8 @@ impl Default for AuditorClientBuilder {
 /// The [`AuditorClient`] handles the interaction with the Auditor instances and allows one to add
 /// records to the database, update records in the database and retrieve the records from the
 /// database.
+///
+/// It should be constructed using the [`AuditorClientBuilder`].
```

Comment:
Minor nitpick: "It is constructed using the..."

Reply:
Fixed